internal/app/apid/main.go

```diff
@@ -26,15 +26,19 @@ import (
 	"github.com/talos-systems/talos/pkg/startup"
 )
 
-var endpoints *string
+var (
+	endpoints       *string
+	useK8sEndpoints *bool
+)
 
 func init() {
 	// Explicitly disable memory profiling to save around 1.4MiB of memory.
 	runtime.MemProfileRate = 0
 
 	log.SetFlags(log.Lshortfile | log.Ldate | log.Lmicroseconds | log.Ltime)
 
-	endpoints = flag.String("endpoints", "", "the IPs of the control plane nodes")
+	endpoints = flag.String("endpoints", "", "the static list of IPs of the control plane nodes")
```

Comment:
Do you know if we support supplying these manually in the config?

Reply:
Nope, we don't, but I thought about it and structured the code so that it's easy to add support for it

Reply:
Yep, exactly. Ok cool. Thanks.